core/ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCard.kt

```diff
@@ -147,20 +148,36 @@ fun NewsResourceCardExpanded(
 fun NewsResourceHeaderImage(
     headerImageUrl: String?,
 ) {
-    AsyncImage(
-        placeholder = if (LocalInspectionMode.current) {
-            painterResource(DesignsystemR.drawable.ic_placeholder_default)
-        } else {
-            // TODO b/228077205, show specific loading image visual
-            null
-        },
+    SubcomposeAsyncImage(
         modifier = Modifier
             .fillMaxWidth()
             .height(180.dp),
         contentScale = ContentScale.Crop,
         model = headerImageUrl,
         // TODO b/226661685: Investigate using alt text of  image to populate content description
-        contentDescription = null, // decorative image
+        contentDescription = null, // decorative image,
+        error = {
+            if (LocalInspectionMode.current) {
+                Image(
+                    painter =
+                    painterResource(DesignsystemR.drawable.ic_placeholder_default),
+                    contentDescription = "placeholder image",
+                )
+            } else {
+                null
```

Comment:
Here we probably need a fallback (error) image.

Reply:
Done, I removed the condition of if (LocalInspectionMode.current) and always returned the ic_placeholder_default, what do you think ?